packages/ilib-lint/docs/plugins_BuiltinPlugin.js.html

```diff
@@ -98,39 +98,39 @@ <h1 class="page-title">plugins/BuiltinPlugin.js</h1>
         description: "Ensure that URLs that appear in the source string are also used in the translated string",
         note: "URL '{matchString}' from the source string does not appear in the target string",
         regexps: [ "((https?|github|ftps?|mailto|file|data|irc):\\/\\/)([\\da-zA-Z\\.-]+)\\.([a-zA-Z\\.]{2,6})([\\/\w\\.-]*)*\\/?" ],
-        link: "https://github.com/ilib-js/ilib-lint/blob/main/docs/resource-url-match.md"
+        link: "https://github.com/ilib-js/ilib-mono/ilib-lint/blob/main/docs/resource-url-match.md"
```

Comment:
This is missing the "packages" part of the path. But, all of the html in the docs directories is generated, so we need to fix the source instead and then run `pnpm doc`.

Reply:
Same as https://github.com/iLib-js/ilib-mono/pull/44#discussion_r1890284014